common/src/main/resources/assets/viabackwards/data/mapping-1.16.2to1.17.json

```diff
@@ -1191,205 +1191,205 @@
-    "block.rooted_dirt.fall": "",
-    "block.rooted_dirt.hit": "",
-    "block.rooted_dirt.place": "",
-    "block.rooted_dirt.step": "",
-    "entity.skeleton.converted_to_stray": "",
-    "block.small_dripleaf.break": "",
-    "block.small_dripleaf.fall": "",
-    "block.small_dripleaf.hit": "",
-    "block.small_dripleaf.place": "",
-    "block.small_dripleaf.step": "",
-    "block.spore_blossom.break": "",
-    "block.spore_blossom.fall": "",
-    "block.spore_blossom.hit": "",
-    "block.spore_blossom.place": "",
-    "block.spore_blossom.step": "",
-    "block.vine.break": "",
-    "block.vine.fall": "",
-    "block.vine.hit": "",
-    "block.vine.place": "",
-    "block.deepslate_bricks.break": "",
-    "block.deepslate_bricks.fall": "",
-    "block.deepslate_bricks.hit": "",
-    "block.deepslate_bricks.place": "",
-    "block.deepslate_bricks.step": "",
-    "block.deepslate.break": "",
-    "block.deepslate.fall": "",
-    "block.deepslate.hit": "",
-    "block.deepslate.place": "",
-    "block.deepslate.step": "",
-    "block.deepslate_tiles.break": "",
-    "block.deepslate_tiles.fall": "",
-    "block.deepslate_tiles.hit": "",
-    "block.deepslate_tiles.place": "",
-    "block.deepslate_tiles.step": "",
-    "block.polished_deepslate.break": "",
-    "block.polished_deepslate.fall": "",
-    "block.polished_deepslate.hit": "",
-    "block.polished_deepslate.place": "",
-    "block.polished_deepslate.step": "",
-    "entity.glow_item_frame.add_item": "",
-    "entity.glow_item_frame.break": "",
-    "entity.glow_item_frame.place": "",
-    "entity.glow_item_frame.remove_item": "",
-    "entity.glow_item_frame.rotate_item": "",
-    "item.axe.scrape": "",
-    "item.axe.wax_off": "",
-    "item.honeycomb.wax_on": "",
-    "entity.goat.ambient": "",
-    "entity.goat.death": "",
-    "entity.goat.hurt": "",
-    "entity.goat.milk": "",
-    "entity.goat.prepare_ram": "",
-    "entity.goat.screaming.ambient": "",
-    "entity.goat.screaming.death": "",
-    "entity.goat.screaming.hurt": "",
-    "entity.goat.screaming.milk": "",
+    "block.sculk_sensor.break": "block.stem.break",
+    "block.sculk_sensor.fall": "block.stem.fall",
+    "block.sculk_sensor.hit": "block.stem.hit",
+    "block.sculk_sensor.place": "block.stem.hit",
+    "block.sculk_sensor.step": "block.stem.step",
+    "entity.axolotl.attack": "entity.fox.hurt",
+    "entity.axolotl.death": "entity.fox.death",
+    "entity.axolotl.hurt": "entity.fox.hurt",
+    "entity.axolotl.idle_air": "entity.elder_guardian.ambient_land",
+    "entity.axolotl.idle_water": "block.bubble_column.bubble_pop",
+    "entity.axolotl.splash": "entity.generic.splash",
+    "entity.axolotl.swim": "entity.generic.swim",
+    "item.bucket.empty_axolotl": "item.bucket.empty",
+    "item.bucket.fill_axolotl": "item.bucket.fill",
+    "item.dye.use": "block.snow.fall",
+    "item.glow_ink_sac.use": "block.beehive.drip",
+    "entity.glow_squid.ambient": "entity.squid.swim",
```

Comment:
The mapped sound doesn't seem to exist judging by the warning on startup

Reply:
I understand how I made this mistake and will fix it.